misk-aws-dynamodb-testing/build.gradle.kts

```diff
@@ -10,11 +10,13 @@ plugins {
 
 dependencies {
   api(Dependencies.awsDynamodb)
+  api(Dependencies.guava)
   api(Dependencies.guice)
   api(Dependencies.javaxInject)
   api(Dependencies.tempestTestingInternal)
   api(project(":misk-aws-dynamodb"))
   api(project(":misk-inject"))
+  implementation(Dependencies.errorproneAnnotations)
```

Comment:
I think we can actually omit these if we want, see: https://github.com/google/guice/pull/1739

Reply:
If we don't need it, let's remove it. 

Could be worth running the dependency analyzer plugin again, last time it caught stuff like this.

Reply:
I pushed a change to omit it, let's see if the tests pass. 🤞